calpip/Task1/Task 1-2.txt

```diff
@@ -1,24 +1,24 @@
 Dear Sir or Madam,
 
 A couple of months ago I bought a long term membership card to one of 
-your gyms located in Winnipeg downtown area. I have paid a considerable 
-high amount of money expecting the best quality of service in return. 
+your gyms located in Winnipeg downtown area. I have paid a considerably 
+high amount of money, expecting the best quality of service in return. 
 Sadly, my experience has been really bad so far.
 
-For one thing, the main reason why I have become a customer was the 
-variety of fitness class announced in your flyers. However, most of 
+For one, the main reason why I became a customer was for the 
+variety of fitness classes announced in your flyers. However, most of 
 those classes, for instance spinning and yoga, simply don’t exist in 
 the gym schedule. In addition, the boxing instructor is absent most of 
-the time and the gym doesn’t provide any replacement for that.
+the time and the gym doesn’t provide a replacement instructor.
```

Comment:
`any replacement` -> `a replacement`
If you expected more than one replacement, you can use `any`. In this case, you only needed one replacement, so it is better to use `a`.

`that` should not be used when describing people, it's better to even use the word `instructor` a second time